ui/eureka/eureka_config.ts

```diff
@@ -44,8 +44,7 @@ UserConfig.registerOptions('eureka', {
       name: {
         en: 'Prefer complete names for Skirmishes/Critical Engagements',
         de: 'Bevorzuge komplette Namen für Scharmützel/Kritische Gefechte',
-        fr:
-          'Préférer les noms complet pour les escarmouches/Affrontements Cruciaux dans Bozja/Zadnor',
+        fr: 'Préférer les noms complet pour les Escarmouches/Affrontements Cruciaux dans Bozja/Zadnor',
```

Comment:
@quisquous, is the line break required?

Reply:
Yeah, if the line is longer than the line length (100 characters) then the line break is required.  That's why the lint step is complaining.  You could also shorten it and remove the Bozja/Zadnor part too.

Reply:
Thank You.